cli/golem-cli/src/command.rs

```diff
@@ -561,6 +561,9 @@ pub enum GolemCliSubcommand {
         /// Optional script_file to run, when defined the repl will execute the script and exit
         #[clap(long, conflicts_with_all = ["script"])]
         script_file: Option<PathBuf>,
+        /// If true (default), the REPL will stream logs from the invoked agents. Can be controlled with the :logs command in the REPL.
+        #[clap(long)]
+        stream: Option<bool>,
```

Comment:
i think all the other "bool" flags now are using `bool` as type, making them behaving only as flags without values. could we keep this consistent and change this flag to something like
```
disable_streaming: bool
no_stream: bool
```